### PR TITLE
chronusdb* ：Add news-module and new features

### PR DESCRIPTION
**Pull request by molliezhang**
### What type of PR is this?

 
/Add Features
/Optimize

### Which issue(s) this PR fixes?

NA

### What this PR does /  why we need it ?

Summary:
1. Add 1.0.6 version features in manual, 
2. Add news module
3. Add version in intro
4. Optimize sentence in intro, manual, quickstart, index
5. Fix the weight in files

### Special notes for your reviewer?
NA

/assign @pethu1110

### CODE REVIEW

File: content/database/chronusdb/intro/version.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+title: "系列和版本"
+description: 本小节主要介绍 时序数据库 ChronusDB 主要系列和版本。 
+keywords: mongodb 版本介绍,系列介绍 
+data: 2021-05-14T00:38:25+09:00
```
Comment (pethu1110) [inline content/database/chronusdb/intro/version.md:5]: 删除date

File: content/database/chronusdb/manual/mgt_connect/mgt_vxnet.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+QingCloud 集群网络环境为独享私有网络，便于集群网络**过滤控制**管理，且不影响其它私有网络环境，确保数据库相对不同业务的网络隔离性。
+
+数据库集群仅可加入已连接路由器的私有网络，且需确保私有网络的 DHCP 处于**打开**状态。 
+
+本小节主要介绍如何切换 ChronusDB 集群私有网络。
+
+## 前提条件
+
+- 已获取 QingCloud 管理工作台登录账号和密码，且已获取集群操作权限。
```
Comment (pethu1110) [inline content/database/chronusdb/manual/mgt_connect/mgt_vxnet.md:19]: QingCloud 管理控制台

File: content/database/chronusdb/news/_index.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+---
+title: "动态与公告"
+date: 2021-04-28T00:38:25+09:00
```
Comment (pethu1110) [inline content/database/chronusdb/news/_index.md:3]: 删除date

File: content/database/chronusdb/quickstart/create_cluster.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+本小节主要介绍如何快速创建 QingCloud ChronusDB 集群。
+
+## 前提条件
+
+- 已获取 QingCloud 管理工作台登录账号和密码，且账号已实名认证。
+- 已获取 ChronusDB 集群操作权限。
+
+## 操作步骤
+
+1. 登录 QingCloud 管理工作台。
```
Comment (pethu1110) [inline content/database/chronusdb/quickstart/create_cluster.md:22]: QingCloud 管理控制台

File: content/database/chronusdb/intro/version.md (hunk 1 above)
Comment (molliezhang) [reply content/database/chronusdb/intro/version.md:5]: done

File: content/database/chronusdb/manual/mgt_connect/mgt_vxnet.md (hunk 2 above)
Comment (molliezhang) [reply content/database/chronusdb/manual/mgt_connect/mgt_vxnet.md:19]: done

File: content/database/chronusdb/news/_index.md (hunk 3 above)
Comment (molliezhang) [reply content/database/chronusdb/news/_index.md:3]: done

File: content/database/chronusdb/quickstart/create_cluster.md (hunk 4 above)
Comment (molliezhang) [reply content/database/chronusdb/quickstart/create_cluster.md:22]: done

**Review by pethu1110 — APPROVED** [review]

**Pull request merged**